src/main/java/Task.java

```diff
@@ -0,0 +1,47 @@
+public class Task {
+    private boolean isCompleted = false;
+    private String taskDescription;
+    private int taskNumber;
+
+    public Task(String taskDescription) {
+        System.out.println("Got it. I've added this task: ");
+        this.taskDescription = taskDescription;
+    }
+
+    public String getTaskDescription() {
+        return this.taskDescription;
+    }
+
+    public void setTaskNumber(int taskNumber) {
+        this.taskNumber = taskNumber;
+    }
+
+    public int getTaskNumber() {
+
+        return this.taskNumber;
+    }
+
+    public void setTaskAsDone() {
+        this.isCompleted = true;
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println("  " + this.toString());
+    }
+
+    public String getTaskType() {
+        String[] taskDescriptionSplitted = this.taskDescription.split(" ");
```

Comment:
Perhaps you could change the name of this array to something that more directly hints that it is an array?